.azure/pipelines/ci.yml

```diff
@@ -659,6 +661,17 @@ stages:
         beforeBuild:
         - powershell: "& ./src/Servers/IIS/tools/UpdateIISExpressCertificate.ps1; & ./src/Servers/IIS/tools/update_schema.ps1"
           displayName: Setup IISExpress test certificates and schema
+        - task: Cache@2
+          inputs:
+            key: 'pw | "$(Agent.OS)" | eng/versions.props'
+            restoreKeys: |
+               pw | "$(Agent.OS)"
```

Comment:
Why fallback to whatever Playwright version has been cached❔ Suggest leaving `restoreKeys` out